src/src/index.html

```diff
@@ -0,0 +1,44 @@
+<!--
+
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta id="theme-color" name="theme-color" content="#FEBC04">
+    <title>Web Starter Kit</title>
+
+    <link rel="manifest" href="/manifest.json">
+    <link rel="icon" href="/images/icons/icon-192x192.png" sizes="192x192" type="image/png">
+    <link rel="icon" href="/images/icons/icon-384x384.png" sizes="384x384" type="image/png">
+
+    <!-- By adding the 'inline' attribute, the build process will inline
+    this CSS file to improve load time performance-->
+    <link rel="stylesheet" href="/styles/index.css" type="text/css" inline>
```

Comment:
To be 100% compliant, should we use `data-inline`? Or is `inline` an official attribute?